src/FSharp.Formatting.CommandTool/BuildCommand.fs

```diff
@@ -643,33 +651,43 @@ type CoreBuildOptions(watch) =
                 if not docsQueued then
                     docsQueued <- true
                     printfn "Detected change in '%s', scheduling rebuild of docs..."  this.input
-                    Async.Start(async {
+                    async {
                         do! Async.Sleep(300)
                         lock monitor (fun () ->
                         docsQueued <- false
                         if runDocContentPhase1() then
                             if runDocContentPhase2() then
                                 regenerateSearchIndex()
-                        ) }) )
+                        )
+                    }
+                    |> Async.RunSynchronously
+                    Serve.signalHotReload <- true
+                ) 
 
             let apiDocsDependenciesChanged = Event<_>()
             apiDocsDependenciesChanged.Publish.Add(fun () ->
                 if not generateQueued then
                     generateQueued <- true
                     printfn "Detected change in built outputs, scheduling rebuild of API docs..."
-                    Async.Start(async {
+                    async {
                         do! Async.Sleep(300)
                         lock monitor (fun () ->
                         generateQueued <- false
                         if runGeneratePhase1() then
                             if runGeneratePhase2() then
-                                regenerateSearchIndex()) }))
+                                regenerateSearchIndex())
+                    }
+                    |> Async.RunSynchronously
+                    Serve.signalHotReload <- true
+                )
 
+            // handler for signalling to refresh the page to the client
+            let hotReloadHandler _ = Serve.signalHotReload <- true
```

Comment:
Does this `hotReloadHandler ` get used anywhere?

Reply:
nope, that is indeed a leftover from a previous test, will remove